types/mparticle__web-sdk/index.d.ts

```diff
@@ -0,0 +1,813 @@
+// Type definitions for mParticle/web-sdk SDK 2.12
+// Project: https://github.com/mParticle/mparticle-web-sdk
+// Definitions by: Alex Sapountzis <https://github.com/asap>
```

Comment:
You're referencing other user, why is this?

Reply:
Alex actually helped me develop this as a typescript project in a hackthon previously which I am finishing up as part of thie PR, so I wanted to give him credit.

Reply:
that's OK, but probably he should be part of PR review, to ACK using Github's handle here.
That's not enforced on DT, but good netiq IMO

Reply:
That makes sense.  Adding him now, as we went through an internal approval process before I posted this PR for Definitely Typed.